community/kernel/src/test/java/org/neo4j/kernel/impl/store/counts/CountsComputerTest.java

```diff
@@ -170,7 +170,7 @@ public void shouldCreateACountsStoreWhenThereAreUnusedRelationshipRecordsInTheDB
         try ( Lifespan life = new Lifespan() )
         {
             CountsTracker store = life.add( new CountsTracker( StringLogger.DEV_NULL, fs, pageCache,
-                    new File( dir, COUNTS_STORE_BASE ) ) );
+                                                               new File( dir, COUNTS_STORE_BASE ) ) );
             assertEquals( BASE_TX_ID + 1 + 1 + 1 + 1 + 1, store.txId() );
 //            assertEquals( 11, store.totalRecordsStored() ); // we do not support yet (label,type,label) counts
```

Comment:
We could remove them. Otherwise the answer here is "no", because (as the comment says) we don't store those counts.